gtsam/nonlinear/PriorFactor.h

```diff
@@ -32,6 +32,10 @@ namespace gtsam {
   public:
     typedef VALUE T;
 
+    // Provide access to the Matrix& version of evaluateError:
+    using NoiseModelFactor1<VALUE>::evaluateError;
```

Comment:
Just a comment that I don't think this is so bad, because usually this won't be called directly anyway (but rather from a `NonlinearFactorGraph`) except in unit tests, and even then it's usually either (a) without the output args or (b) using factorTesting macros, so I don't think it will actually affect too much old code.